os/arch/arm/src/imxrt/imxrt_norflash.c

```diff
@@ -774,6 +774,9 @@ status_t imxrt_flexspi_nor_flash_erase_sector(FLEXSPI_Type *base, uint32_t addre
 	}
 
 	status = imxrt_flexspi_nor_wait_bus_busy(base);
+
```

Comment:
both of imxrt_norflash.c and imxrt_hyperflash.c need this code.

Reply:
Let me add this to imxrt_hyperflash.c later after testing imxrt1050 configs and TCs.

Reply:
got it.